common/lib/xmodule/xmodule/course_module.py

```diff
@@ -742,7 +742,7 @@ class CourseFields(object):
     allow_public_wiki_access = Boolean(
         display_name=_("Allow Public Wiki Access"),
         help=_(
-            "Enter true or false. If true, edX users can view the course wiki even "
+            "Enter true or false. If true, students can view the course wiki even "
```

Comment:
I prefer the generic "users" over "students", though to be fair, that could be years of conditioning at Stanford where part of our fork was not calling people "students", but rather "learners" or "users" ;)